.travis.yml

```diff
@@ -46,6 +46,9 @@ before_script:
       esac ;
       set +x;
     fi
+
+    # pull python3-rados from distro packages
+    sudo apt install -y python3-rados
```

Comment:
This doesn't have any effect, please remove this. The package is maintained inside the Docker container used for testing.